node/src/components/deploy_acceptor.rs

```diff
@@ -62,6 +62,10 @@ pub(crate) enum Error {
         prestate_hash: Digest,
         failure: DeployParameterFailure,
     },
+
+    /// The deploy received by the node from the client has expired.
+    #[error("deploy received by the node from the client has expired")]
```

Comment:
Let's mention deploy TTL somewhere - it should make it easier for the clients to understand what they did wrong and what to fix.

Reply:
The error message now includes the timestamp of expiry to let the user know when their deploy expired. @goral09 , is that okay?

Reply:
And the timestamp of the node when it checked the expiry